packages/itwinui-react/src/core/Tile/TileNew.tsx

```diff
@@ -0,0 +1,646 @@
+  /**
+   * Whether the tile is expected to be interactable (i.e. `onClick`).
+   * It becomes focusable and gets on hover styling.
+   */
+  isActionable?: boolean;
+  /**
+   * Display a loading state.
+   * @default false
+   */
+  isLoading?: boolean;
+  /**
+   * Flag whether the tile is disabled.
+   *
+   * Note: This only affects the tile. You need to manually disable
+   * the buttons and other interactive elements inside the tile.
+   *
+   * @default false
+   */
+  isDisabled?: boolean;
+  setActionable: React.Dispatch<React.SetStateAction<boolean>>;
+  onClick?: React.MouseEventHandler<HTMLElement>;
+};
+
+// ----------------------------------------------------------------------------
+// Tile.ThumbnailArea component
+
+const TileThumbnailArea = polymorphic.div('iui-tile-thumbnail');
+TileThumbnailArea.displayName = 'TileNew.ThumbnailArea';
+
+type TileThumbnailAreaOwnProps = {}; // eslint-disable-line @typescript-eslint/ban-types
+
+// ----------------------------------------------------------------------------
+// Tile.ThumbnailPicture component
+
+const TileThumbnailPicture = React.forwardRef((props, ref) => {
+  const { as: Element = 'div', className, children, ...rest } = props;
+  return (
+    <Element
+      className={cx('iui-tile-thumbnail-picture', className)}
+      style={{
+        backgroundImage:
+          children && typeof children === 'string'
+            ? `url(${children})`
+            : undefined,
+      }}
+      ref={ref}
+      {...rest}
+    />
+  );
+}) as PolymorphicForwardRefComponent<'div', TileThumbnailPictureOwnProps>;
+
+TileThumbnailPicture.displayName = 'TileNew.TileThumbnailPicture';
+
+type TileThumbnailPictureOwnProps = {}; // eslint-disable-line @typescript-eslint/ban-types
+
+// ----------------------------------------------------------------------------
+// Tile.ThumbnailAvatar component
+
+const TileThumbnailAvatar = React.forwardRef((props, ref) => {
+  const { as: Element = 'div', className, children, ...rest } = props;
+  if (React.isValidElement(children)) {
+    return React.createElement(children.type, {
+      ...children.props,
+      className: cx('iui-thumbnail-icon', children.props.className),
+    });
+  }
+
+  return (
+    <Element className={cx(className)} ref={ref} {...rest}>
+      {children}
+    </Element>
+  );
+}) as PolymorphicForwardRefComponent<'div', TileThumbnailAvatarOwnProps>;
+
+TileThumbnailPicture.displayName = 'TileNew.TileThumbnailAvatar';
+
+type TileThumbnailAvatarOwnProps = {}; // eslint-disable-line @typescript-eslint/ban-types
+
+// ----------------------------------------------------------------------------
+// Tile.QuickAction component
+
+const TileQuickAction = React.forwardRef((props, ref) => {
+  const { as: Element = 'button', className, children, ...rest } = props;
+  return (
+    //todo: styling is all off, need to fix after
+    <Element
+      className={cx('iui-button', 'iui-tile-thumbnail-quick-action', className)}
```

Comment:
This could be fixed when styling issues are fixed, but I'll comment anyway

If `Element` isn't a `button`, do we still want to add `iui-button` styling? The same goes for `TypeIndicator`

Reply:
This has been updated, it is now whatever <Box/> is and no more `iui-button` classname, there is a new subcomponent for icon which is polymographically default to be `button` with`iui-button-icon`, seems to be working as intended.